sandbox/src/game.ts

```diff
@@ -37,30 +37,39 @@ var logger = ex.Logger.getInstance();
 logger.defaultLevel = ex.LogLevel.Debug;
 
 // Create an the game container
+ex.Flags.enable('use-webgl');
 var game = new ex.Engine({
-  width: 800,
-  height: 600,
-  antialiasing: false,
+  width: 800 / 2,
+  height: 600 / 2,
+  viewport: { width: 800, height: 600 },
   canvasElementId: 'game',
   suppressHiDPIScaling: false,
-  suppressPlayButton: true
+  suppressPlayButton: true,
+  antialiasing: false,
+  snapToPixel: true
 });
+game.setAntialiasing(false);
```

Comment:
Is this needed when using `antialiasing: false`?

Reply:
It is not